odoo/models.py

```diff
@@ -3469,6 +3469,12 @@ def get_base_url(self):
             raise ValueError("Expected singleton or no record: %s" % self)
         return self.env['ir.config_parameter'].sudo().get_param('web.base.url')
 
+    def _get_html_link(self, title=None):
```

Comment:
Note that this is a bit strange to have that defined in odoo but supported only with mail JS. Otherwise it has no real use outside of discuss. Would probably define in mail ?

Reply:
Wasn't exactly sure whether it was supported only in chatter or more globally in `web`, I'll move it in `mail` then :)

Reply:
Ah, maybe it is now supported by web, now you talk about it ...  It was discuss only initially as far as I remember. But you know, old people, ...